src/classes/EscalationTrigger.cls

```diff
@@ -102,6 +102,34 @@ public with sharing class EscalationTrigger extends BaseTrigger {
 	}
 
 	/**
+	* Auto populate the rh_escal__EscalationCase__c	record if primary case on the escalation is not null.
+	*/
+	@TestVisible
+	private void createEscalationCaseWhenPrimaryCaseIsNotNull() {
+		if (this.isDelete || this.isUpdate) {
```

Comment:
Make this

```java
if (!this.isInsert) {
```

It's unlikely that we'd add support for `undelete` or another type, but we want to be specific that this is only for inserts.